native/cocos/bindings/event/EventDispatcher.h

```diff
@@ -212,19 +212,19 @@ class KeyboardEvent : public OSEvent {
     bool   shiftKeyActive = false;
     // TODO(mingo): support caps lock?
 };
-
+union ccUnionArg {
```

Comment:
- please do not start with lowercase characters
- please use more meaningful type name, such as `UnionEventParameterType`

Reply:
use `EventParameterType` instead